backend/app/infrastructure/database/models.py

```diff
@@ -110,6 +120,16 @@ class FinalizedEpochSnapshot(BaseModel):
     leftover = Column(db.String, nullable=False)
     withdrawals_merkle_root = Column(db.String)
     total_withdrawals = Column(db.String)
+    ppf = Column(db.String, nullable=True)  # TBC is it needed?
```

Comment:
I think it's not needed if it's already stored in a pending snapshot. These values won't change through the allocation window

Reply:
Okay, will get a rid of it.